Tools/TestWebKitAPI/cocoa/TestUIDelegate.mm

```diff
@@ -97,6 +113,43 @@ - (NSString *)waitForAlert
     return result.autorelease();
 }
 
+- (NSString *)waitForConfirm
+{
+    EXPECT_FALSE(self.runJavaScriptConfirmPanelWithMessage);
+
+    __block bool finished = false;
+    __block RetainPtr<NSString> result;
+    self.runJavaScriptConfirmPanelWithMessage = ^(WKWebView *, NSString *message, WKFrameInfo *, void (^completionHandler)(BOOL)) {
+        result = message;
+        finished = true;
+        completionHandler(YES);
+    };
+
+    TestWebKitAPI::Util::run(&finished);
+
+    self.runJavaScriptConfirmPanelWithMessage = nil;
+    return result.autorelease();
+}
+
+- (NSString *)waitForPromptWithDefaultInput:(NSString *)expectedDefaultInput
```

Comment:
I'd call this "Reply" or "Result" instead of "DefaultInput"
Also drop "expected"
The return value of this is "prompt" not "result"